Voeg initiele versie voor Open API van RVVA toe

TimvdLippe opened the pull request:
Fixes #221

TimvdLippe committed:
Voeg initiele versie voor Open API van RVVA toe

ChielTimmermans commented on media/rvva-openapi.json:
```diff
@@ -0,0 +1,305 @@
+            "description": "The latest valid version of ProcessingActivityVersion",
+            "type": "number"
+          },
+          "processing_activity_id": {
```
I would just call this `id` instead of `processing_activity_id.
Otherwise it feels like a stutter: ProcessingActivity.processing_activity_id

TimvdLippe committed:
Fix code review comments

Nil-NMB01 commented:
Wanneer is het de bedoeling dat we deze mergen @TimvdLippe

TimvdLippe commented:
Zodra #221 is uitgevoerd, waarschijnlijk tijdens het traject van de Inzage app.

TimvdLippe committed:
Update media/rvva-openapi.json

Co-authored-by: Nil Netra Barua <nil.barua@logius.nl>

TimvdLippe committed:
Update openapi.json

TimvdLippe committed:
Voeg applicable laws toe aan LegalBasis

Hiermee is het mogelijk om ook aan te geven welke specifieke
wetten van toepassing zijn. Bij de overige legal basis waarden is
dit niet van toepassing en volstaat enkel de enum string.

TimvdLippe committed:
Fix capitalization
